command/core.py

```diff
@@ -28,3 +30,15 @@ def handle_app_mention(self, event_data):
                 return 1
             except KeyError:
                 return -1
+
+    def handle_member_join(self, event_data):
+        """Handle the event of a new user joining the workspace."""
+        new_user_id = event_data["event"]["user"]["id"]
+        new_user = User(new_user_id)
+        self.__facade.store_user(new_user)
+        welcome = 'Welcome to Lauchpad!'
+        try:
+            self.__bot.send_dm(welcome, new_user_id)
+            return True
```

Comment:
A small thing - for the other handlers it looks like you used 1 and -1, but here it's using `True` and `False`. We should probably be consistent, though I also know we have #68 to get rid of these anyway.